### PR TITLE
Issue #753 form element styles (simple - pt1)

### PR DESCRIPTION
**Pull request by Graham-72**
Implement form element styles as per the style guide
backdrop/backdrop-issues/issues/753

### CODE REVIEW

File: core/themes/seven/css/style.css [hunk 1]
```diff
@@ -789,16 +789,11 @@ input.form-search,
 input.form-file,
 textarea.form-textarea,
 select.form-select {
```
Comment (jenlampton) [inline core/themes/seven/css/style.css]: It looks like these changes are affecting the select element as well.  Select element styles should be handled in https://github.com/backdrop/backdrop-issues/issues/754

File: core/themes/seven/css/style.css [hunk 2]
```diff
@@ -789,16 +789,11 @@ input.form-search,
...
-  background: #fff;
-  color: #333;
-  -webkit-transition: border linear 0.2s, box-shadow linear 0.2s;
-  -moz-transition: border linear 0.2s, box-shadow linear 0.2s;
-  transition: border linear 0.2s, box-shadow linear 0.2s;
+  color: #000;
+  background: none repeat scroll 0% 0% transparent;
+  border-radius: 4px;
+  border: 3px solid rgba(255, 255, 255, 0);
+  box-shadow: 0px 0px 0px 2px #D0D0D0;
```
Comment (jenlampton) [inline core/themes/seven/css/style.css:796]: Looks like we may be missing the 10px padding.

File: core/themes/seven/css/style.css [hunk 3]
```diff
@@ -789,16 +789,11 @@ input.form-search,
...
-  border-top-color: #999;
-  background: #fff;
-  color: #333;
-  -webkit-transition: border linear 0.2s, box-shadow linear 0.2s;
-  -moz-transition: border linear 0.2s, box-shadow linear 0.2s;
-  transition: border linear 0.2s, box-shadow linear 0.2s;
+  color: #000;
+  background: none repeat scroll 0% 0% transparent;
+  border-radius: 4px;
+  border: 3px solid rgba(255, 255, 255, 0);
```
Comment (jenlampton) [inline core/themes/seven/css/style.css:795]: border should be 2px not 3px, perhaps a typo ;)

**Comment by jenlampton** [comment]
@Graham-72 thanks for all your amazing work so far!

**Comment by jenlampton** [comment]
I'll see if I can sort out some of the differences here and get this ready to go.  
Also, since PRs are merged as they are, the commits should be clean and ready to go. Please note our standards for commit messages: https://backdropcms.org/develop/pull-requests They should be a single sentance (one line) starting with the issue number, and ending with a peroid. For example:

```
Issue #xxx: Thing that this does, and maybe why it was needed.
```

**Comment by jenlampton** [comment]
I've replaced this with a new PR over at https://github.com/backdrop/backdrop/pull/841

**Comment by Graham-72** [comment]
@jenlampton - I am now a bit puzzled because of your comment

> border should be 2px not 3px, perhaps a typo ;)

I have been copying and pasting from the styleguide style.css but maybe not from the current version? I have been downloading this from http://backdrop-ops.github.io/styleguide/core/themes/seven/style.css
Is there some other way I should use to keep up-to-date? (or perhaps the typo is in that source)

Some of my other PRs seem to have picked up worse mistakes and I'm not sure how :worried:

**Comment by jenlampton** [comment]
@Graham-72  I'm not sure there's a better way to stay up to date.

I've been viewing source on http://backdrop-ops.github.io/styleguide.  I wouldn't necessarily trust what's in the style.css file line-by-line because it's likely to be overridden several times (even in that one file alone) before it gets to the browser. That project has been a bit of a free-for-all in terms of code, we've been using it just to get things to look right, the code in there is not the best. It is quite possible that the typo is in that file.

In general, I think our process is sound - this is why we have reviews and reviewers :) Not only do we benefit from your code (it doesn't have to be perfect - it still helps!) but we can also get you all trained on git / GitHub / PRs and everything else. In the end, everyone wins, right? :)

**Pull request closed without merging**